src/scheduled_tasks.rs

```diff
@@ -122,6 +122,11 @@ fn update_banned_when_expired(conn: &mut PgConnection) {
   sql_query(update_ban_expires_stmt)
     .execute(conn)
     .expect("update banned when expires");
+
+  let delete_community_ban_expires_stmt = "delete from community_person_ban where expires < now()";
```

Comment:
Rather than checking the `community_person_ban.expires` column every time, I correctly delete the rows which are expired anyway in this scheduled job.

Reply:
Makes sense, you can do the same for site bans. Btw why are queries in this file all written as plain strings and not using dsl?

Reply:
Good call, I fixed the ones that I could.